proxy-robots-module/src/main/java/orwell/proxy/robot/RobotElementStateVisitor.java

```diff
@@ -0,0 +1,63 @@
+package orwell.proxy.robot;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import orwell.messages.Robot;
+
+
+/**
+ * Created by Michaël Ludmann on 5/12/15.
+ */
+public class RobotElementStateVisitor implements IRobotElementVisitor {
```

Comment:
So something like : RobotElementBuildStateVisitor ?
Would make more sense, but this starts to get a bit too long...